packages/smart-wallet/src/smartWallet.js

```diff
@@ -135,7 +134,7 @@ const mapToRecord = map => Object.fromEntries(map.entries());
  * @returns {State}
  */
 export const initState = (unique, shared) => {
-  // Some validation of inputs. "any" erefs because this synchronous call can't check more than that.
```

Comment:
why remove this explanation for `M.any`? 

btw, should some of these `M.remotable`?

Reply:
> why remove this explanation for `M.any`?

Because I got rid of all the `M.any` cases. Oops... but then I reverted that change without putting the comment back.

> btw, should some of these `M.remotable`?

Yes.

Reply:
Refining the typeguards isn't necessary to fix #6652, so arguably it should go in a separate PR. You don't mind if I include it here, do you?

Reply:
SGTM, makes sense with the anys gone :)